types/react-draft-wysiwyg/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": false,
```

Comment:
This should be `true` for all new packages.

Reply:
This still isn't set to true

Reply:
This still isn't set to true